ap/settings/common.py

```diff
@@ -81,6 +82,29 @@
     "django_structlog.middlewares.RequestMiddleware",
 ]
 
+# -- Sentry error tracking
+
+if os.environ.get("SENTRY_DSN"):
+    SENTRY_ENVIRONMENT = ENV
+    KUBERNETES_ENV = "EKS"
```

Comment:
Not sure if we really need this? Presume its from control panel back from when it wasnt on EKS

Reply:
@jacobwoffenden what do you think?

Reply:
Agreed